draft-ietf-rats-corim.md

```diff
@@ -542,7 +541,7 @@ Described in {{sec-common-validity}}.
 
 # Concise Module Identifier (CoMID) {#sec-comid}
 
-A CoMID tag contains information about hardware, firmware, or module composition.
+A CoMID tag contains claims and metadata about hardware, firmware, or module composition.
```

Comment:
not sure what "metadata" adds on top of "claims"

```suggestion
A CoMID tag contains claims about hardware, firmware, or module composition.
```

Reply:
The claims are encoded in a measurement values map, but metadata about the claims include items like `entities` and `linked-tags`

Reply:
The metadata is about the CoMID itself rather than _"HW, FW or module composition."_

Reply:
Also we have claims inside measurement-values-map and mkey can contain some additional information about the claim, like measurement identifier and extensible objects about claims!

Reply:
I prefer to remain generic at this point in the document.  "Information" (as in the original) seems more than appropriate.

Reply:
i am happy with information as well! Lets change it then!

Reply:
However, let us discuss this in the meeting, as others have already approved this version of definition!

Reply:
Information is fine by me